apps/pm-user/src/components/organisms/sideBar/SideBar.js

```diff
@@ -1,17 +1,21 @@
 import React from 'react';
-
 import { Menu } from '../../atoms';
+import { ProfileSummery } from "../../molecules";
 import styles from './sideBar.module.scss';
 import { DEFAULT_SELECTED_ITEM_KEY, MENU_ITEMS } from './constants';
 
 function SideBar() {
-  const handleMenuItemClick = ({ key }) => {
 
+  const handleMenuItemClick = ({ key }) => {
+    //some code here
   };
 
+
   return (
     <div className={styles.container}>
-      <div className={styles.profileSummary}></div>
+    <div className={styles.profileSummary}>
+        <ProfileSummery imageChangeLink="#" userDetails="Software Developer, 32" userImageSrc="https://zos.alipayobjects.com/rmsportal/jkjgkEfvpUPVyRjUImniVslZfWPnJuuZ.png" userName="Vinit Sharma" />
+      </div>
       <div className={styles.menu}>
         <Menu
           defaultSelectedKeys={[DEFAULT_SELECTED_ITEM_KEY]}
```

Comment:
prop-types is not used in this component. Please add it.

Reply:
here propTypes are not needed. because there is no props in the sideB
ar